repos/re-theme/src/styleInjector/useStyleTag.js

```diff
@@ -93,7 +93,7 @@ export const useStyleTag = (style, className='', config) => {
     const { blocks, filtered } = convertToCss(style, config)
 
     // Create a unique selector based on the className and built blocks
-    const selector = getSelector(className, blocks.join(''))
+    const selector = getSelector(className, blocks.join(''), 'keg')
```

Comment:
It would be better if this could be defined outside the context of Re-Theme
* We could put it in the theme.RTMeta object, and you could call the `useTheme` hook here to get access to it.
* Or maybe we expose a method similar to the setDefaultTheme, that sets a default prefix
* That way it could be customizable

It's no blocking, and works for now, but can you create a ticket in the Jira backlog to a add that.

Reply:
will do.
ticket here: https://jira.simpleviewtools.com/browse/ZEN-433

Reply:
Sweet thanks